Saucier720-app/src/app/recipes/FEC/recipe-card/recipe-card.component.ts

```diff
@@ -70,15 +72,24 @@ export class RecipeCardComponent implements OnInit {
     }
   }
 
-  getAuthorCreditFromRecipeID(recipeID: string): string {
+  public getAuthorCreditFromRecipeID(recipeID: string): string {
+    // used to get recipe author from recipeID
     const author = recipeID.replace(/\d+/g, '');
-  
     if (author === 'json') {
       return 'MealDealz Classic Recipe';
     } else {
       return 'Created by ' + author;
     }
   }
+
+  public isCurrentUserRecipe(recipeID: string): boolean {
+    // used to check if current recipe is made my the current user
+    if (recipeID.replace(/\d+/g, '') === this.cookieService.get("sessionID").replace(/\d+/g, '')){
```

Comment:
This just checks if it's not a premade recipe? Jason Recipe?

Reply:
correct